### PR TITLE
Changed code to keep all variables in the input dataset.

### LINKED ISSUE #61: Typo in Tendril object element
Tendrils objects have an element named 'AEfreqTreshold'.  Suggest that this is renamed to 'AEfreqThreshold' for v1.0.

### PR DESCRIPTION
**Pull request by mark-edmondson-jones**
Also corrected spelling of Threshold.

Closes #61 #62

### CODE REVIEW

File: R/Tendril.R [hunk 1]
```diff
@@ -207,24 +251,24 @@ Tendril <- function(mydata,
...
-                       p.adj = tendril.retval$data$p.adj,
-                       fish = tendril.retval$data$fish,
-                       rdiff = tendril.retval$data$rdiff,
-                       RR = tendril.retval$data$RR,
-                       OR = tendril.retval$data$OR,
-                       FDR.tot = tendril.retval$data$FDR.tot,
-                       TermsCount = tendril.retval$data$TermsCount
-    )
-    tendril.retval$data <- data
+#    data <- data.frame(Unique.Subject.Identifier = as.character(tendril.retval$data$Unique.Subject.Identifier),
```
Comment (stefanoborini) [inline R/Tendril.R]: Don't comment out code. We can always recover code by going back. If it's no longer relevant, remove it.

File: R/formatting.R [hunk 2]
```diff
@@ -45,9 +37,9 @@ joinData <- function(dataset1, dataset2, Tag){
   dataset1$col.id<-as.numeric(dataset1$col.id)
   dataset2$col.id<-as.numeric(dataset2$col.id)
   joined <- dplyr::left_join(dataset1, dataset2, by = c("col.id" = "col.id"))
-  joined<-joined[-c(11, seq(17,21,1))]
-  colnames(joined)<- c("StartDay", "Unique.Subject.Identifier", "Terms", "Treat", "rot.factor", "col.id",
-                   "old.day", "mod", "dir", "k", "angsum", "cx", "x", "y", "ang")
+#  joined<-joined[-c(11, seq(17,21,1))]
```
Comment (stefanoborini) [inline R/formatting.R]: same

**Comment by stefanoborini** [comment]
Which issue is this addressing?

File: R/formatting.R (hunk 2 above)
Comment (mark-edmondson-jones) [reply R/formatting.R]: same

**Comment by mark-edmondson-jones** [comment]
> Which issue is this addressing?

61 & 62

**Comment by stefanoborini** [comment]
Why did you close the PR?

**Comment by mark-edmondson-jones** [comment]
Replaced by #66 which removed commented out code.

**Comment by stefanoborini** [comment]
You can keep committing to an open PR

**Pull request closed without merging**